package.json

```diff
@@ -75,7 +75,7 @@
     "yargs": "^17.3.1"
   },
   "dependencies": {
-    "@microsoft/powerplatform-cli-wrapper": "0.1.82",
+    "@microsoft/powerplatform-cli-wrapper": "0.1.83",
```

Comment:
PR is missing the update to the lock file. Pls run `npm install` locally and add the package.lock.json changes